[proj] build failure

**Host Environment**
- OS: Windows 10 20H2 19042.1706
- Compiler: Visual Studio 2019 (Build engine 16.11.2.50704)

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install proj:x64-windows`

**Failure logs**
```
Computing installation plan...
The following packages will be built and installed:
    proj[core,net,tiff]:x64-windows -> 9.0.1
Detecting compiler hash for triplet "x64-windows"...
-- Using HTTP(S)_PROXY in environment variables.
Restored 0 packages from D:\cschulte\AppData\Local\vcpkg\archives in 241 us. Use --debug to see more details.
Installing 1/1 proj:x64-windows...
Building proj[core,net,tiff]:x64-windows...
-- Using cached OSGeo-PROJ-9.0.1.tar.gz.
-- Cleaning sources at D:/cschulte/DevEnv64/vcpkg/buildtrees/proj/src/9.0.1-b314c5518d.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/cschulte/DevEnv64/vcpkg/downloads/OSGeo-PROJ-9.0.1.tar.gz
-- Applying patch fix-win-output-name.patch
-- Applying patch fix-proj4-targets-cmake.patch
-- Applying patch tools-cmake.patch
-- Using source at D:/cschulte/DevEnv64/vcpkg/buildtrees/proj/src/9.0.1-b314c5518d.clean
-- Found external ninja('1.10.2').
-- Configuring x64-windows
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:129 (message):
    Command failed: D:/cschulte/DevEnv64/vcpkg/downloads/tools/cmake-3.22.2-windows/cmake-3.22.2-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j9
    Working Directory: D:/cschulte/DevEnv64/vcpkg/buildtrees/proj/x64-windows-dbg
    See logs for more information:
      D:\cschulte\DevEnv64\vcpkg\buildtrees\proj\install-x64-windows-dbg-out.log
      D:\cschulte\DevEnv64\vcpkg\buildtrees\proj\install-x64-windows-dbg-err.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/proj/portfile.cmake:41 (vcpkg_cmake_install)
  scripts/ports.cmake:146 (include)


error: building proj:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+proj
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[proj]+Build+error
Include '[proj] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
    vcpkg-tool version: 2022-06-17-9268e366206712e38102b28dbd1617697a99ff2e
    vcpkg-scripts version: 019027ada 2022-06-21 (8 days ago)

You can also use the prefilled template from D:\cschulte\DevEnv64\vcpkg\installed\vcpkg\issue_body.md.
```
**Additional context**
This issue does not appear with the previous version of proj. Indeed from version 9.0.0 to 9.0.1 man install path has been modified from share/man/man1 to $CMAKE_INSTALL_MANDIR/man1 (https://github.com/OSGeo/PROJ/commit/1461c9b30d93941e858cf68c01117b83f095c3d6). I clearly have the needed privileges on the vcpkg folder as every other install works correctly. As he tries to create path /man1 I don't know where he is trying to create this folder.
[install-x64-windows-dbg-out.log](https://github.com/microsoft/vcpkg/files/9017349/install-x64-windows-dbg-out.log)
[install-x64-windows-dbg-err.log](https://github.com/microsoft/vcpkg/files/9017350/install-x64-windows-dbg-err.log)

--- a/ports/proj/portfile.cmake
+++ b/ports/proj/portfile.cmake
@@ -8,6 +8,7 @@ vcpkg_from_github(
         fix-win-output-name.patch
         fix-proj4-targets-cmake.patch
         tools-cmake.patch
+        remove-doc.patch
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS

--- a/ports/proj/remove-doc.patch
+++ b/ports/proj/remove-doc.patch
@@ -1,0 +1,34 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 4c9744e6..a6d5e4eb 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -318,15 +335,12 @@ include_directories(${PROJ_SOURCE_DIR}/src)
+ add_subdirectory(data)
+ add_subdirectory(include)
+ add_subdirectory(src)
+-add_subdirectory(man)
+ add_subdirectory(cmake)
+ if(BUILD_TESTING)
+   add_subdirectory(test)
+ endif()
+ 
+-set(docfiles COPYING NEWS AUTHORS)
+-install(FILES ${docfiles}
+-        DESTINATION "${DOCDIR}")
++
+ 
+ ################################################################################
+ # pkg-config support
+@@ -378,6 +392,7 @@ set(CPACK_SOURCE_IGNORE_FILES
+ 
+ include(CPack)
+ 
++if(0)
+ # Simplify README.md to README
+ add_custom_target(README
+   COMMAND ${CMAKE_COMMAND}
+@@ -393,3 +408,4 @@ if(NOT _is_multi_config_generator)
+   )
+   message(STATUS "PROJ: Configured 'dist' target")
+ endif()
++endif()

--- a/ports/proj/vcpkg.json
+++ b/ports/proj/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "proj",
   "version": "9.0.1",
+  "port-version": 1,
   "description": "PROJ library for cartographic projections",
   "homepage": "https://proj.org/",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5618,7 +5618,7 @@
     },
     "proj": {
       "baseline": "9.0.1",
-      "port-version": 0
+      "port-version": 1
     },
     "proj4": {
       "baseline": "8.9.9",

--- a/versions/p-/proj.json
+++ b/versions/p-/proj.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "d533a2d58a17e3074d52ec045604b32ed049eacb",
+      "version": "9.0.1",
+      "port-version": 1
+    },
+    {
       "git-tree": "55c13547b4c812fc8bbb32a09260b3e54bc0a352",
       "version": "9.0.1",
       "port-version": 0

